io.github.lemmygtk.lemoa.json

```diff
@@ -0,0 +1,48 @@
+{
+    "id": "io.github.lemmygtk.lemoa",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "44",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable",
+        "org.freedesktop.Sdk.Extension.llvm14"
+    ],
+    "command": "lemoa",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--device=dri",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--env=RUST_LOG=lemoa=debug",
+        "--env=G_MESSAGES_DEBUG=none",
+        "--env=RUST_BACKTRACE=1"
+    ],
+    "build-options": {
+        "append-path": "/usr/lib/sdk/rust-stable/bin:/usr/lib/sdk/llvm14/bin",
+        "env": {
+            "CARGO_REGISTRIES_CRATES_IO_PROTOCOL": "sparse",
+            "CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_LINKER": "clang",
+            "CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_RUSTFLAGS": "-C link-arg=-fuse-ld=/usr/lib/sdk/rust-stable/bin/mold",
+            "CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER": "clang",
+            "CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_RUSTFLAGS": "-C link-arg=-fuse-ld=/usr/lib/sdk/rust-stable/bin/mold"
+        }
+    },
+    "modules": [
+        {
+            "name": "lemoa",
+            "buildsystem": "meson",
+            "builddir": true,
+            "config-opts": [
+                "-Dprofile=default"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "path": "https://github.com/lemmygtk/lemoa.git",
```

Comment:
this build will no work as it is done offline.

Reply:
you need to either generate a source list with the `flatpak-cargo-generator` https://github.com/flatpak/flatpak-builder-tools/tree/master/cargo or use a tarball with vendored crates.

Reply:
So we should add
```
"build-args": [
            "--share=network"
        ],
```
?

Reply:
no. this is not allowed either. 

I mentionned the two possible solutions.

Reply:
Sorry, I didn't yet see your message when answering

Reply:
Should I open an issue at the `flatpak-builder-tools` repo?